packages/react-devtools-shared/src/__tests__/ComponentSettings-test.js

```diff
@@ -0,0 +1,102 @@
+import React from 'react';
+import { render, unmountComponentAtNode } from 'react-dom';
+import ComponentsSettings from '../devtools/views/Settings/ComponentsSettings';
+
+describe('ComponentsSettings', () => {
+  let container;
+
+  beforeEach(() => {
+    // Set up a DOM element as a render target
+    container = document.createElement('div');
+    document.body.appendChild(container);
+  });
+
+  afterEach(() => {
+    // Clean up on exiting
+    unmountComponentAtNode(container);
+    container.remove();
+    container = null;
+  });
+
+  // @reactVersion >= 17
```

Comment:
Please remove these annotations, my change has landed on `main`, so these tests should be included in tests runs now

Reply:
yup i was going to remove it eventually!